manifests/o/Oven-sh/Bun/Baseline/1.1.0/Oven-sh.Bun.Baseline.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Oven-sh.Bun.Baseline
+PackageVersion: 1.1.0
+PackageLocale: en-US
+Publisher: Oven-sh
+PublisherUrl: https://github.com/oven-sh
+PublisherSupportUrl: https://github.com/oven-sh/bun/issues
+PackageName: Bun (Baseline build)
```

Comment:
With how WinGet normalizes package names, info inside the parenthesis may get ignored when performing package correlation. WinGet may map an installation of `Oven-sh.Bun.Baseline` to `Oven-sh.Bun`. I would suggest having a PackageName like `Bun Baseline` that is written to ARP

Reply:
https://github.com/microsoft/winget-cli/issues/4201#issuecomment-1964694857 After revisiting the comment, I think the 'ignoring whatever inside parenthesis' only affects `AppsAndFeaturesEntries > DisplayName` and not PackageName. But since we control the name written to Registry in case of portables, I would still stay on the safe side and drop the parenthesis